pom.xml

```diff
@@ -54,6 +54,29 @@
             <artifactId>postgresql</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-security</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-api</artifactId>
+            <version>0.11.2</version>
+        </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-impl</artifactId>
+            <version>0.11.2</version>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-jackson</artifactId>
+            <version>0.11.2</version>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
Please remove repetitive dependencies

Reply:
jjwt-impl contains the core implementation of the JWT library. It provides the basic functionality for creating, parsing, and validating JSON Web Tokens.
jjwt-jackson provides support for integrating the JWT library with Jackson, which is a popular JSON library in Java. It includes Jackson serializers and deserializers for JWT-related classes, allowing you to easily convert JWTs to JSON and vice versa.
That is why i have used both these dependencies

Reply:
Removing any of the dependency leads to an exception which tell unable to  load class respective of their use which result in that in response of authenticate it is not able to generate token.